app-modules/student-data-model/src/Models/Student.php

```diff
@@ -118,9 +117,7 @@ class Student extends BaseAuthenticatable implements Auditable, Subscribable, Ed
     public function __construct(array $attributes = [])
     {
         parent::__construct($attributes);
-
-        // Set the $timestamps property based on the feature flag
-        $this->timestamps = Feature::active('student_timestamp_fields');
+        $this->timestamps = true;
```

Comment:
I think we can remove this `__construct` now as well as the `$timestamp` property. Which should make the model default back to having timestamps. Try that out and ensure it has the timestamps created.